src/Tests/AzureClientTests.cs

```diff
@@ -198,6 +198,7 @@ public void TestManualTargets()
         [DataRow("--clear", "FullComputation", ExecuteStatus.Ok)]
         [DataRow("honeywell.mock", "FullComputation", ExecuteStatus.Error)]
         [DataRow("honeywell.mock", "BasicMeasurementFeedback", ExecuteStatus.Ok)]
+        [DataRow("quantinuum.mock", "AdaptiveExecution", ExecuteStatus.Ok)]
```

Comment:
What exactly is the test coverage that we have with this? I don't think this necessarily checks that QIR is submitted, does it? What test fails without the code changes in AzureClient.cs and AzureExecutionTarget.cs?

Reply:
These tests check local usage of the IQ# kernel, for checking the QIR that gets submitted, I'd suggest adding to the live tests as well.

Reply:
By live tests you mean tests that are not on this repo? Where are these?

Reply:
They are in this repo, under the `tests.live/` folder, but those tests aren't run with the e2e pipeline — there's a separate pipeline that runs for live tests that pulls those integration tests in. As of 5601aeb4bb58dd47e32ae1cbb1571e69e93940e5, this PR now includes additional integration tests in that folder that ensure that teleportation runs successfully when the target is set to a Quantinuum API validator and when the target capability is `AdaptiveExecution`.